### PR TITLE
Implement Flow Component

### PR DESCRIPTION
**Pull request by cat-mandres**

### CODE REVIEW

File: src/clr-addons/flow-bar/flow-bar.scss [hunk 1]
```diff
@@ -0,0 +1,30 @@
+@include exports('flow-bar.clr-addons') {
+  clr-flow-bar {
```
Comment (Wallabeng) [inline src/clr-addons/flow-bar/flow-bar.scss]: please  use a class instead of tag

File: src/clr-addons/flow-bar/flow-bar.ts [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  @Input('clrSteps') _steps: FlowBarStep[] = [];
+  @Input('clrActiveStep') _activeStep: FlowBarStep;
+
+  @Output('clrActiveStepChanged') _activeStepChanged: EventEmitter<FlowBarStep> = new EventEmitter(false);
+
+  ngOnInit() {
+    // If no active step is set as input, select the first one
+    // If the active step input is not enabled, select the first one
+    if (!this._activeStep || !this._activeStep.enabled) {
+      this._activeStep = this._steps[0];
```
Comment (Wallabeng) [inline src/clr-addons/flow-bar/flow-bar.ts]: consider the case that the first step is disabled

File: src/dev/src/app/flow-bar/flow-bar.demo.ts [hunk 3]
```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2018 Porsche Informatik. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import { Component, OnInit, ViewChild } from '@angular/core';
+import { ClrFlowBar, FlowBarStep } from '../../../../clr-addons/flow-bar';
+
+@Component({
+  selector: 'clr-pager-demo',
```
Comment (Wallabeng) [inline src/dev/src/app/flow-bar/flow-bar.demo.ts]: please change selector

File: src/clr-addons/flow-bar/flow-bar.ts [hunk 4]
```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2018 Porsche Informatik. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+
+export interface FlowBarStep {
```
Comment (Wallabeng) [inline src/clr-addons/flow-bar/flow-bar.ts]: prefix with Clr, as its public

File: src/clr-addons/components.clr-addons.scss [hunk 5]
```diff
@@ -5,37 +5,60 @@
...
+  padding: 0.5rem 1rem 0 1rem;
   display: flex;
-  flex-shrink: 0;
-}
+  flex: 0 0 auto;
 
-.content-header > h2 {
-  margin-top: 0;
-}
+  h2 {
```
Comment (Wallabeng) [inline src/clr-addons/components.clr-addons.scss]: we should not overwrite h2 globally, as also h2's in content will get their margin-top set to 0

File: src/clr-addons/flow-bar/flow-bar.ts [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+
+  set activeStep(step: ClrFlowBarStep) {
+    this._activeStep = step;
+    this._activeStepChange.emit(this._activeStep);
+  }
+
+  ngOnInit() {
+    // If no active step is set as input or the active step is not enabled, select the first enabled step
+    if (!this._activeStep || !this._activeStep.enabled) {
+      this._activeStep = this._steps.find(step => {
```
Comment (Wallabeng) [inline src/clr-addons/flow-bar/flow-bar.ts]: the initial step should not emit the "activeStepChange" event? just asking to be sure

File: src/dev/src/app/flow-bar-content-panel/flow-bar-content-panel.demo.ts [hunk 7]
```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2018 Porsche Informatik. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import { Component } from '@angular/core';
+import { ClrFlowBarStep } from '../../../../clr-addons/flow-bar';
```
Comment (Wallabeng) [inline src/dev/src/app/flow-bar-content-panel/flow-bar-content-panel.demo.ts]: use @porscheinformatik/clr-addons import here

File: src/clr-addons/components.clr-addons.scss (hunk 5 above)
Comment (cat-mandres) [reply src/clr-addons/components.clr-addons.scss]: this is nested inside .content-header

**Pull request merged**